web/components/PetPostModal/PetPostModal.tsx

```diff
@@ -38,10 +38,10 @@ import {
   fosterTypeDescriptions,
 } from "../../utils/types/post";
 import { trpc } from "../../utils/trpc";
-import { ObjectId } from "mongoose";
 import { useState } from "react";
 import { z } from "zod";
 import { useAuth } from "../../context/auth";
+import { Schema } from "mongoose";
```

Comment:
Sorry, I should've been more clear! We want
```js
import { Types } from "mongoose";
Types.ObjectId
```
 This is different from `Schema.Types.ObjectId`: https://mongoosejs.com/docs/schematypes.html#what-is-a-schematype